kubernetes/keycloaks.keycloak.org-v1.yml

```diff
@@ -2674,9 +2674,13 @@ spec:
                   A secret containing the TLS configuration for HTTPS. Reference: https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets.
                   The special value `INSECURE-DISABLE` disables https.
                 type: string
-              disableDefaultIngress:
-                description: Disable the default ingress.
-                type: boolean
+              ingress:
+                description: |-
+                  The deployment is, by default, exposed through a basic nginx ingress.
```

Comment:
Can we assume Nginx ingress controller is always installed/used? Or best is to just remove `nginx` from the description?

Reply:
Good point! This comes from the docs  and obviously is a mistake. We can't assume nginx controller is the default here.

Reply:
Thanks for the review, @pedroigor. I fixed it.